events/events.go

```diff
@@ -84,6 +84,8 @@ type DataConnect struct {
 	TargetServerName           string
 	Error                      error
 	IsVirtualLocation          bool
+	Upload                     uint64
+	Download                   uint64
```

Comment:
Why was this added here and in `DataDisconnect` now? Is this part of some analytics task? :thinking: